webpack.index-worker.config.js

```diff
@@ -0,0 +1,42 @@
+const path = require('path');
+
+const config = {
+  mode: 'development',
+  entry: './src/index-worker.ts',
+  target: "electron-renderer",
+  devtool: 'inline-source-map',
```

Comment:
that's the reason we only got 10% out of it..

Reply:
Look at the end of the file